content/en/monitoring/notification-manager.md

```diff
@@ -66,13 +66,15 @@ Send auditing alerts to Notification Manager:
         "group_interval": "30s"
 ```
 
+> These configurations are the default configuration. If you do not want to receive a certain type of alert, you can delete the corresponding configuration.
```

Comment:
The above is the default configuration.

Reply:
yes

Reply:
`These configurations are the default configuration` => `The above is the default configuration`